src/v2i-hub/CDA1TenthPlugin/src/WebSocketServer.h

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (C) 2025 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+#pragma once
+#include <libwebsockets.h>
```

Comment:
I think this is fine but in the future we should try to reuse the C++ websocket library we already have in V2X Hub. This C library is only used by the CommandPlugin and we were looking to remove it with boosts web socket library used in the Pedestrian plugin. Before merging this to develop, this maybe something we want to consider. But for CDA1Tenth feature branch this is fine.